### PR TITLE
E2E: Update e2e java selenium tests according new behaviour on UD

### PR DESCRIPTION
**Pull request by artaleks9**
# What does this PR do?
* Add the _AddWorkspaceToOrganizationTest_, _ShareWorkspaceMemberTest_ to _UNDER_REPAIR_ group related to known issue #15822
* Update locator of web element according to change on UD
* Update the _RecreateUpdateStrategyTest_ according to change behaviour on UD

### What issues does this PR fix or reference?
#15811
#15821
#15822

### CODE REVIEW

File: tests/legacy-e2e/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/organization/AddWorkspaceToOrganizationTest.java [hunk 1]
```diff
@@ -33,7 +33,9 @@
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
-@Test(groups = {TestGroup.MULTIUSER, TestGroup.OPENSHIFT, TestGroup.K8S})
+// Known permanent failure https://github.com/eclipse/che/issues/15822
+@Test(groups = {UNDER_REPAIR})
+// @Test(groups = {TestGroup.MULTIUSER, TestGroup.OPENSHIFT, TestGroup.K8S})
```
Comment (dmytro-ndp) [inline tests/legacy-e2e/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/organization/AddWorkspaceToOrganizationTest.java]: We just need to add `TestGroup.UNDER_REPAIR` to existed annotation @Test.

File: tests/legacy-e2e/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/organization/ShareWorkspaceMemberTest.java [hunk 2]
```diff
@@ -43,7 +43,9 @@
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
-@Test(groups = {TestGroup.MULTIUSER, TestGroup.OPENSHIFT, TestGroup.K8S})
+// Known permanent failure https://github.com/eclipse/che/issues/15822
+@Test(groups = {UNDER_REPAIR})
+// @Test(groups = {TestGroup.MULTIUSER, TestGroup.OPENSHIFT, TestGroup.K8S})
```
Comment (dmytro-ndp) [inline tests/legacy-e2e/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/organization/ShareWorkspaceMemberTest.java]: We just need to add `TestGroup.UNDER_REPAIR` to existed annotation @Test.

File: tests/legacy-e2e/che-selenium-test/src/test/java/org/eclipse/che/selenium/hotupdate/recreate/RecreateUpdateStrategyTest.java [hunk 3]
```diff
@@ -80,12 +79,6 @@ public void checkRecreateUpdateStrategy() throws Exception {
 
     cheTestSystemClient.stop();
 
-    // open the Dashboard and make sure that workspace is not available after suspending system
```
Comment (dmytro-ndp) [inline tests/legacy-e2e/che-selenium-test/src/test/java/org/eclipse/che/selenium/hotupdate/recreate/RecreateUpdateStrategyTest.java:83]: This verification looks important, as for me. Could you, please, explain why do we need to remove this step?

File: tests/legacy-e2e/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/organization/ShareWorkspaceMemberTest.java (hunk 2 above)
Comment (artaleks9) [reply tests/legacy-e2e/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/organization/ShareWorkspaceMemberTest.java]: Fixed.

File: tests/legacy-e2e/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/organization/AddWorkspaceToOrganizationTest.java (hunk 1 above)
Comment (artaleks9) [reply tests/legacy-e2e/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/organization/AddWorkspaceToOrganizationTest.java]: Fixed.

File: tests/legacy-e2e/che-selenium-test/src/test/java/org/eclipse/che/selenium/hotupdate/recreate/RecreateUpdateStrategyTest.java (hunk 3 above)
Comment (artaleks9) [reply tests/legacy-e2e/che-selenium-test/src/test/java/org/eclipse/che/selenium/hotupdate/recreate/RecreateUpdateStrategyTest.java:83]: Now the behaviour is changed. After stop `che-server` by API the dashboard is not open more and there is present only `che loader` page. 
I had conversation with @olexii4  (who made changes in the behaviour) and actually both variants are not ideal and require discussion.
But it is not a bug and we should continue checking this functionality as is.

Comment (dmytro-ndp) [reply tests/legacy-e2e/che-selenium-test/src/test/java/org/eclipse/che/selenium/hotupdate/recreate/RecreateUpdateStrategyTest.java:83]: Got it. Thanks for explanation!

**Review by dmytro-ndp — APPROVED** [review]

**Pull request merged**